Std/Data/Array/Basic.lean

```diff
@@ -36,6 +36,18 @@ Sort an array using `compare` to compare elements.
 def qsortOrd [ord : Ord α] (xs : Array α) : Array α :=
   xs.qsort λ x y => compare x y |>.isLT
 
+set_option linter.unusedVariables.funArgs false in
+/--
+Returns the first minimal element among `d` and elements of the array.
+If `start` and `stop` are given, only the subarray `xs[start:stop]` is
+considered (in addition to `d`).
+-/
+@[inline]
+protected def minWith [ord : Ord α]
```

Comment:
```suggestion
/--
Returns the first minimal element among `d` and elements of the array.
If `start` and `stop` are given, only the subarray `xs[start:stop]` is
considered (in addition to `d`).
-/
@[inline]
protected def minWith [Ord α]
```
No need to name typeclasses.

Reply:
Oh, I missed that you were using the `(ord := ...)` named argument syntax below.

An alternative is
```
  @Array.minWith _ ord.opposite xs d start stop
```

but I'm ambivalent about which is better. Feel free to just click "resolve" without any action if you prefer!

Reply:
I like your suggestion more TBH, but I think this might be an externally visible API change. So I think we should be consistent in the whole file about this whether we want these to be named or not. We should probably treat this in a separate issue/PR.